pkg/web/router/router.go

```diff
@@ -68,7 +68,7 @@ func SetAppRoutes(r *echo.Group, appsHandler apps.HTTPHandler) {
 	//     description: Invalid id supplied
 	//   404:
 	//     description: App not found
```

Comment:
IMHO will be go add in the descriptions that this router and apps just return active apps

Reply:
so `Active App not found` suggests there may be an active app when there may not be, I think `App not found` should be enough for a customer to see on the endpoint

Reply:
@davidffrench,

Should we really do it or instead of all this change just add in the service layer to return in the response one error telling that the app searched is inactive? Wdyt? @austincunningham

Reply:
Are there currently multiple errors returned?

It should just be the one error of `app not found`.

I might be missing the ask here.

Reply:
My suggestion was an app return app not found or app disable when it is disabled or the app.
instead of the not found for both scenarios.